modules/browser_object_tabbar.py

```diff
@@ -65,12 +65,29 @@ def new_window_by_keys(self, sys_platform: str) -> BasePage:
             ).perform()
         return self
 
+    def reopen_closed_tab_by_keys(self, sys_platform: str) -> BasePage:
+        """Use keyboard shortcut to reopen a last closed tab"""
+        if sys_platform == "Darwin":
+            self.actions.key_down(Keys.COMMAND).key_down(Keys.SHIFT).send_keys(
+                "t"
+            ).key_up(Keys.SHIFT).key_up(Keys.COMMAND).perform()
+        else:
+            self.actions.key_down(Keys.CONTROL).key_down(Keys.SHIFT).send_keys(
+                "t"
+            ).key_up(Keys.SHIFT).key_up(Keys.CONTROL).perform()
+        return self
+
     def click_tab_by_title(self, title: str) -> BasePage:
         """Given a full page title, click the corresponding tab"""
         with self.driver.context(self.driver.CONTEXT_CHROME):
             self.get_element("tab-by-title", labels=[title]).click()
         return self
 
+    def get_tab_by_title(self, title: str) -> BasePage:
+        """Given a full page title, return the corresponding tab"""
+        with self.driver.context(self.driver.CONTEXT_CHROME):
```

Comment:
I know all the other methods set the context manually, but try it without! I want to see if it works... (it should)

Reply:
The behaviour of tabs in the private browser is so confusing: when trying to reopen a tab I get this failure:
```
FAILED tests/security_and_privacy/test_undo_close_tab_private_browsing.py::test_undo_close_tab_private_browsing - selenium.common.exceptions.NoSuchWindowException: Message: Browsing context has been discarded
```

otherwise everything works as expected. For my CI failure, I'm going to start an interactive task and see whats going